synapse/tests/test_model_inet.py

```diff
@@ -1584,3 +1584,30 @@ def test_model_inet_urlredir(self):
 
             self.eq(node[1].get('inet:urlredir:dst'), 'http://bar.com/')
             self.eq(node[1].get('inet:urlredir:dst:fqdn'), 'bar.com')
+
+    def test_model_inet_rfc2822_addr(self):
+
+        with self.getRamCore() as core:
+
+            n0 = core.formTufoByProp('inet:rfc2822:addr', 'FooBar')
```

Comment:
Is this actually a valid address????

Reply:
Sigh...  yes, but not an "email address".  The rfc 2822 thing includes an arbitrary identifier thing.  :(